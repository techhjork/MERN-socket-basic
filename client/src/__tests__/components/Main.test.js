import React from 'react';
import { shallow } from 'enzyme';
import Main from '../../components/Main';


test('Should render Main component correctly', () => {
    const wrapper = shallow(<Main />);
    expect(wrapper).toMatchSnapshot();
});
