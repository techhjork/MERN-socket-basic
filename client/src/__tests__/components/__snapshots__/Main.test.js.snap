// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Main component correctly 1`] = `
<div>
  <h1>
    Welcome to react start app
  </h1>
</div>
`;
